Automattic/wp-calypso
# Plans: Replace CRM Entrepreneur with CRM Free

#### Changes proposed in this Pull Request

Related to `1196108640073826-as-1199527862646257`.

* Add an extra `isFree` property to the `SelectorProductCost` type, to indicate products that should be displayed as Free.
* Create new monthly and yearly product slugs for Jetpack CRM Free, so they can be looked up and handled on the Plans page like any other product.
* Move the price markup in `JetpackProductCardAlt2` into a separate component inside the same file, to modularize things a bit and to support displaying "Free" as a price.
* Add a style rule for "Free" pricing so that it matches other product cards.
* Replace CRM Entrepreneur with CRM Free in the product grid ordering.

#### Implementation notes

I'm not a huge fan of creating new product slugs and objects for CRM Free, but this seemed the most obvious way to accommodate a product with no price while leaving CRM Entrepreneur alone in case we need it later. Please let me know if there's a better way!

#### Testing instructions

* In Calypso Blue, Green, and/or Jetpack Connect, visit the Plans/Pricing page.
* Verify that the CRM card is visible in the product grid in the second row, next to Site Search and Anti-spam.
* Verify all copy is correct, including description, call-to-action, and feature line items. (For now, it should all be the same as for the pre-existing CRM Entrepreneur product.)
* Verify that clicking the call-to-action button sends you to `jetpackcrm.com/pricing`.

#### Reference screenshots

<img width="1078" alt="image" src="https://user-images.githubusercontent.com/670067/107791686-d92ac180-6d19-11eb-8556-8504d4cf10f2.png">

<img width="1089" alt="image" src="https://user-images.githubusercontent.com/670067/107790964-02971d80-6d19-11eb-9b8b-f14a89b8a5da.png">
